wicket-native-websocket/wicket-native-websocket-javax/src/main/java/org/apache/wicket/protocol/ws/javax/WicketEndpoint.java

```diff
@@ -43,6 +46,118 @@
  */
 public class WicketEndpoint extends Endpoint
 {
+	private static class JavaxWebSocketSessionWrapper  implements IWebSocketSession
```

Comment:
Let's move this class as a non-inner class with package-private visibility.
Also its name could be `JavaxWebSocketSession` to be more consistent with the other classes in -javax module.

Reply:
done